zeppelin-web-angular/src/app/pages/workspace/notebook/action-bar/action-bar.component.less

```
@@ -113,3 +113,50 @@
     }
   }
 });
+
+::ng-deep .search-code {
```

```suggestion
::ng-deep html.dark .search-code,
::ng-deep .search-code {
```
The newly added dark-mode styles are overriding our `search-code` styles. Let's add a selector so these styles apply in both themes.